core/src/main/java/io/aiven/klaw/service/SchemaRegistrySyncControllerService.java

```diff
@@ -143,7 +148,9 @@ private List<SchemaSubjectInfoResponse> getSchemasFromDb(
         manageDatabase
             .getHandleDbRequests()
             .getTopicAndVersionsForEnvAndTenantId(schemaEnvId, tenantId);
-
+    if (log.isDebugEnabled()) {
```

Comment:
Yes if there is an object being passed into the logging statement it will get processed by the logger and then it will look and see that it is in fact in debug mode and not print it out.
This saves that ObjectMapper object->string conversion from happening every time.